SwiftyAttributes/Sources/iOS_tvOS/UIKit+SwiftyAttributes.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UIKit+SwiftyAttributes.swift
+//  SwiftyAttributes
+//
+//  Created by Roman Podymov on 26/05/19.
+//  Copyright © 2019 Roman Podymov. All rights reserved.
+//
+
+#if os(macOS)
+#elseif os(watchOS)
+#else
+    import UIKit
+    public typealias NavigationBar = UINavigationBar
+#endif
+
+#if os(macOS)
+#elseif os(watchOS)
+#else
```

Comment:
I think you could just extend the above macros for the entire file so that you don't need to repeat them